apps/src/lib/kits/maker/ui/DiscountAdminOverride.jsx

```diff
@@ -0,0 +1,141 @@
+/** @file Admin Override for Maker Discount Code Eligibility*/
+import React, {Component, PropTypes} from 'react';
+import i18n from "@cdo/locale";
+import SchoolAutocompleteDropdown from '../../../../templates/SchoolAutocompleteDropdown';
+import Button from "../../../../templates/Button";
+import ValidationStep, {Status} from '../../../../lib/ui/ValidationStep';
```

Comment:
I personally really don't like seeing these paths with lots of `../` in them. I would prefer to see an absolute reference instead, i.e. `from '@cdo/apps/lib/ui/validationStep';`.

That said, there's some argument in favor in the relative paths in that I think some peoples' editors know how to follow them, but not our absolute paths.

Reply:
Yeah, I don't love the inconsistency on this. My editor can follow the "../" paths, but @ cdo is much cleaner. I think I might try to stick with @ cdo though.